docs/source/galleries/how_to/01_preprocess_a_session.py

```diff
@@ -46,9 +46,10 @@
 if __name__ == "__main__":
 
     pp_steps = {
-        "1": ["phase_shift", {}],  
-        "2": ["bandpass_filter", {"freq_min": 300, "freq_max": 6000}],
-        "3": ["common_reference", {"operator": "median", "reference": "global"}],
+    "1": ["phase_shift", {}],
+    "2": ["bandpass_filter", {"freq_min": 300, "freq_max": 6000}],
+    "3": ["common_reference", {"operator": "median"}],
+    "4": ["whiten", {"apply_mean": True, "mode": "global", "dtype": "float32"}]  # ✅ Added whiten properly
```

Comment:
In this case, we will not highlight the whitening in the example scripts. The reason for this is quite obscure, basically many sorters run the whitening step internally. We could however run it on the spikeinterface side and turn it off in the sorter, although it is not clear (to me at least) how much this might effect the results. In theory spikeinterface whiten should be the same as the kilosort whiten but it is not easy to test (against the matlab versions). For ease of example it makes sense to show only some basic preprocessing steps here.